src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+
+}
+
+.block {
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  line-height: 300px;
+  color: white;
+}
+
+.block-1 {
+  background-color: hsl(0, 100%, 50%);
+}
+
+.block-2 {
+  background-color: hsl(0, 100%, 40%);
+}
+
+.block-3 {
+  background-color: hsl(0, 100%, 30%);
+}
+
+.block-4 {
+  background-color: hsl(0, 100%, 20%);
+}
+
+.block-5 {
+  background-color: hsl(0, 100%, 10%);
+}
+
+.block-6 {
+  background-color: hsl(0, 100%, 0%);
+}
+
+@media (min-width: 600px) and (max-width: 900px) {
+  .block-3 {
+    order: 1;
+  }
+  .block-5 {
+    order: 1;
+  }
+  .block-6 {
+    order: 1;
+  }
+}
+
+@media (min-width: 900px) {
+  .block-6 {
+    order: 1;
+  }
+  .block-5 {
+    order: 2;
+  }
+  .block-4 {
+    order: 3;
+  }
+}
+
+@media (min-width: 1200px) {
+  .block {
+    flex-basis: 400px;
+  }
+}
```

Comment:
```suggestion
```
In fact, there's no need in this, if you'll specify flex-basis for each of media-queries